Makefile

```diff
@@ -46,6 +49,32 @@ gendeepcopy:
 	  -O zz_generated.deepcopy \
 	  -h boilerplate.go.txt
 
+STATIC_API_DIRS = k8s.io/apimachinery/pkg/apis/meta/v1
+STATIC_API_DIRS += k8s.io/apimachinery/pkg/api/resource
+STATIC_API_DIRS += k8s.io/apimachinery/pkg/version
+STATIC_API_DIRS += k8s.io/apimachinery/pkg/runtime
+STATIC_API_DIRS += k8s.io/apimachinery/pkg/util/intstr
+STATIC_API_DIRS += k8s.io/api/core/v1
+
+# Automatically extract vendored apis under vendor/k8s.io/api.
+VENDOR_API_DIRS = $(shell find vendor/k8s.io/api -type d | grep -E 'v\d+(alpha\d+|beta\d+)*' | sed -e 's/^vendor\///')
```

Comment:
This will be evaluated every time, so I think you want to use `:=` (although it probably doesn't much matter yet)

e.g. https://github.com/kubernetes/kops/commit/49f60b383cddbad68bb1001a886b2c7e00b6e0ce#diff-b67911656ef5d18c4ae36cb6741b7965

Reply:
done.